src/finance/components/TariffsManagement.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <Breadcrumb :home="home" :model="items">
+    <template #item="{ item }">
+      <router-link :to="item.to" custom v-slot="{ route, navigate, isActive, isExactActive }">
+        <a @click="navigate" :class="{ 'active-link': isActive, 'active-link-exact': isExactActive }">{{
+          route.meta.title
+        }}</a>
+      </router-link>
+    </template>
+  </Breadcrumb>
+  <div class="container">
+    <h1 class="page-title">{{ title }}</h1>
```

Comment:
you can use title from router like so: route.meta.title. Instead of using local title variable. It's not an issues, just saying

Reply:
Thanks, this is good advice.